### DISCUSSION TITLE
Submit your site power by Hargo Hugo

### DISCUSSION THREAD

**Discussion by shampaakter** [gethugothemes/hargo-hugo#33] [Show and tell]
Thanks for creating your site with Hargo  Hugo. Let's share your website to inspire others. You can share your website link here, we will add it to the list.

**Comment by vibhoragarwal** [comment]
Hello Folks
Thanks for providing an awesome theme.
Here I built : https://studiorisha.com

Request to add, also request you to check if all is well with licensing at the footer ?
(I am not a web designer but just managed to put it based on your thorough example)

One potential minor improvement - multiple images on details page can auto-move just like testimonials.
